client/src/partials/cash/modals/selectCashbox.modal.js

```diff
@@ -0,0 +1,57 @@
+angular.module('bhima.controllers')
+.controller('SelectCashboxModalController', SelectCashboxModalController);
+
+SelectCashboxModalController.$inject = ['SessionService', '$uibModalInstance', 'CashboxService', '$location', 'AppCache', 'cashboxId'];
```

Comment:
This line is very long.  In the future, could you use the newlines to break it up?  Like this:
```js
SelectCashboxModalController.$inject = [
  'SessionService', '$uibModalInstance', 'CashboxService', '$location', 'AppCache', 'cashboxId'
];
```